src/models/hook.js

```diff
@@ -622,7 +628,7 @@ const Hook = DataConnected.extend({
   },
 
   getEntity() {
-    return this._space[this.spaceRef || this._parent.getSpace()[0]];
+    return this._space[this.spaceRef] || this._parent._space[this.spaceRef] || this._parent._space[this._parent.getSpace()[0]];
```

Comment:
this is ok now but it will go away when we remove `hook.spaceRef` in favor of `hook.space`
we will need `hook.space` for other features

Reply:
Agree